rules/REQUEST-942-APPLICATION-ATTACK-SQLI.conf

```diff
@@ -742,7 +748,7 @@ SecRule ARGS_NAMES|ARGS|XML:/* "@rx (?i)[\s\v\"'-\)`]*?\b([0-9A-Z_a-z]+)\b[\s\v\
 #
 # -=[ SQL Function Names ]=-
 #
-# This rule has one less stricter sibling: 942151.
+# This rule has one less strict sibling: 942151.
```

Comment:
The wording here is a bit confusing. In all other cases we have documented the least strict rule and labelled the other rules as "stricter siblings". I would do the same here: treat 942151 as the base rule and the others as stricter siblings.